osivalidator/osi_general_validator.py

```diff
@@ -14,7 +14,7 @@
     import osi_rules
     import osi_validator_logger
     import osi_rules_checker
-    import osi_trace
+    from format.OSITrace import OSITrace
 except Exception as e:
     print(
         "Make sure you have installed the requirements with 'pip install -r requirements.txt'!"
```

Comment:
Filename is outdated. We have splitted into requirements needed for develping and the ones for running the software.

Reply:
But this is part of the open-simulation-interface python package now. What do you mean by "Filename is outdated"?

Reply:
Maybe I am confused but that looks like the try except for the import of dependedncies for the osi-validation and not osi (sub module) with the failing message to install the requirements.txt (which does not exist in osi-validation)?

Reply:
Ah sorry, your comment was about the error message. I thought it was about line 17. My mistake. Yes, I'll fix the message.